packages/app/src/components/DBInfraPanel.tsx

```diff
@@ -119,7 +119,7 @@ const InfraSubpanelGroup = ({
                       where,
                       groupBy: [],
                       aggFn: 'avg',
-                      field: `${fieldPrefix}cpu.utilization - Gauge`,
```

Comment:
this should be fixed by https://github.com/hyperdxio/hyperdx/commit/43e32aafc7819b0015d4e24bff5b2fe4a88f3517